src/index.js

```diff
@@ -0,0 +1,127 @@
+const updateCity = () => {
+  const userInput = document.querySelector('#user_input').value;
+  document.getElementById('city_name').innerHTML = userInput;
+};
+
+const createResetListener = () => {
+  const resetButton = document.getElementById('reset');
+  resetButton.addEventListener('click', function(){
```

Comment:
For the callback functions in each of the `addEventListener` functions, consider encapsulating that functionality in a named function rather than an anonymous function to enhance readability.